From 795069a61237d881703a481bc8adc699ebbc7601 Mon Sep 17 00:00:00 2001
From: Richard Zhu <hongxing.zhu@nxp.com>
Date: Fri, 14 Sep 2018 16:30:40 +0800
Subject: [PATCH 4653/5242] MLK-19558-2 ARM64: dts: enable multi-vdev on one
 rpmsg channel

commit  14cad6fb9e8e652194c7824c2bc072939424a672 from
https://source.codeaurora.org/external/imx/linux-imx.git

Support the multi-vdev on one rpmsg channel on 8qm/qxp mek boards.

Signed-off-by: Richard Zhu <hongxing.zhu@nxp.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 arch/arm64/boot/dts/freescale/fsl-imx8qm-mek.dtsi |    6 +++---
 arch/arm64/boot/dts/freescale/fsl-imx8qxp-mek.dts |    4 ++--
 2 files changed, 5 insertions(+), 5 deletions(-)

diff --git a/arch/arm64/boot/dts/freescale/fsl-imx8qm-mek.dtsi b/arch/arm64/boot/dts/freescale/fsl-imx8qm-mek.dtsi
index 1aff741..0b324ec 100644
--- a/arch/arm64/boot/dts/freescale/fsl-imx8qm-mek.dtsi
+++ b/arch/arm64/boot/dts/freescale/fsl-imx8qm-mek.dtsi
@@ -1093,8 +1093,8 @@
 	/*
 	 * 64K for one rpmsg instance:
 	 */
-	vdev-nums = <1>;
-	reg = <0x0 0x90000000 0x0 0x10000>;
+	vdev-nums = <2>;
+	reg = <0x0 0x90000000 0x0 0x20000>;
 	status = "okay";
 };
 
@@ -1111,7 +1111,7 @@
 	 * 64K for one rpmsg instance:
 	 */
 	vdev-nums = <1>;
-	reg = <0x0 0x90100000 0x0 0x10000>;
+	reg = <0x0 0x90100000 0x0 0x20000>;
 	status = "okay";
 };
 
diff --git a/arch/arm64/boot/dts/freescale/fsl-imx8qxp-mek.dts b/arch/arm64/boot/dts/freescale/fsl-imx8qxp-mek.dts
index 2670775..b1b50bd 100755
--- a/arch/arm64/boot/dts/freescale/fsl-imx8qxp-mek.dts
+++ b/arch/arm64/boot/dts/freescale/fsl-imx8qxp-mek.dts
@@ -973,8 +973,8 @@
 	/*
 	 * 64K for one rpmsg instance:
 	 */
-	vdev-nums = <1>;
-	reg = <0x0 0x90000000 0x0 0x10000>;
+	vdev-nums = <2>;
+	reg = <0x0 0x90000000 0x0 0x20000>;
 	status = "okay";
 };
 
-- 
1.7.9.5

